Tests/Tests/CampaignsListManagerSpec.swift

```diff
@@ -34,6 +34,16 @@ class CampaignsListManagerSpec: QuickSpec {
 
                 context("and service error has occured") {
 
+                    beforeEach {
+                        Constants.Retry.Tests.setInitialDelayMS(1000)
+                        Constants.Retry.Tests.setBackOffUpperBoundSeconds(1)
+                    }
+
+                    afterEach {
+                        Constants.Retry.Tests.setInitialDelayMS(10000)
```

Comment:
Do these need to be set both before **and** after?

Reply:
In `after` I'm just setting delays back to their original value.